test/acceptance/features/bindAppToService.feature

```diff
@@ -723,3 +723,47 @@ Feature: Bind an application to a service
         Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-cross-ns-service" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-cross-ns-service" should be changed to "True"
         And The application env var "BACKEND_HOST_CROSS_NS_SERVICE" has value "cross.ns.service.stable.example.com"
+
+    Scenario: Bind an application to a openshift route
```

Comment:
Please place the scenario into a separate feature (`verifyExamples.feature`) and tag it with `@example` tag. Thx

Reply:
OKAY.

Reply:
You are correct in general but then you need to define what a "feature" actually is. IMHO "bind app to service" is way too broad, look:
```feature
Feature: Bind an application to a service

    As a user of Service Binding Operator
    I want to bind applications to services it depends on
...
```

and it's basically everything what the operator does. Originally, we started with a single feature file for simplicity, but now when we have almost 50 scenarios (and number is growing with almost every PR) we need to start splitting the files logically if possible, because:
1. we need to keep the size of feature files reasonably large to be able to manage them.
2. it's good to have related scenarios grouped - like we have in case of annotations.
I agree that In this particular case "demonstrating and tutoring how to use the operator" (what is the point of the examples) is not a "feature of the operator" per say, but rather we use the concept of feature and use it to group scenarios that are related not by a feature of the operator itself but by purpose (to verify examples).

Reply:
Okay, I'll make the changes.

Reply:
@pmacik I've updated it PTAL.